samples/meetings-content-bubble/csharp/ContentBubble/Bots/ContentBubbleBot.cs

```diff
@@ -34,21 +42,57 @@ public ContentBubbleBot(IConfiguration configuration)
                                         new AgendaItem { Topic = "Increase research budget by 10%" , Id = 2},
                                         new AgendaItem { Topic = "Continue with WFH for next 3 months" , Id = 3}},
             };
+            _httpClientFactory = httpClientFactory;
         }
         protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
One line space before it & also add method summary

Reply:
Added